File: Assignment1_1.py
#Program to divide two numbers

def DivisionOfTwoNumbers(iValue1,iValue2):
    
    fAns = 0.0
    
    if(iValue2 == 0):
        print("Invalid Input")
        return -1
    
    fAns = iValue1/iValue2
    return fAns
    

def main():
    
    iNo1 = 0
    iNo2 = 0
    fRet = 0.0
    
    print("Enter the first number:")
    iNo1 = int(input())
    
    print("Enter the Second Number:")
    iNo2 = int(input())
    
    fRet = DivisionOfTwoNumbers(iNo1,iNo2)
    
    print("Division is : ",fRet)
    
if __name__=="__main__":
    main()

File: Assignment1_1.cpp
//Write a program to divide the two numbers

#include<iostream>
using namespace std;

class Division
{
    public:
        int iNo1;
        int iNo2;
        float fAns;

        Division(int A,int B,float C = 0.0f)
        {
            iNo1 = A;
            iNo2 = B;
            fAns = C;
        }

        void DivideTwoNumbers(int iNo1,int iNo2)
        {
            if(iNo2 == 0)
            {
                cout<<"Invalid Input...\n";
            }

            fAns = (float)iNo1/(float)iNo2;
            cout<<"Division of Two Numbers is:"<<fAns<<"\n";
        }
};

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    cout<<"Enter the First Number:"<<"\n";
    cin>>iValue1;

    cout<<"Enter the Second Number:"<<"\n";
    cin>>iValue2;

    Division dobj1(iValue1,iValue2);

    dobj1.DivideTwoNumbers(iValue1,iValue2);
    
    return 0;
}